### PR TITLE
FOCUS #330: Pricing Category value updates

### PR DESCRIPTION
**Pull request by flanakin**
Updates PricingCategory values:
- "On-Demand" to "Standard"
- "Commitment-Based" to "Committed"

Related to #330

### CODE REVIEW

File: supporting_content/columns/pricingcategory.md [hunk 1]
```diff
@@ -14,68 +14,69 @@ Current column mappings found in available data sets:
...
+| Provider  | Data set                    | Provider value     | PricingCategory | PricingSubcategory |
+| --------- | --------------------------- | ------------------ | --------------- | ------------------ |
+| AWS       | CUR (PurchaseOption)        | On-Demand          | Standard        | (depends on usage) |
+| AWS       | CUR (PurchaseOption)        | Reserved Instances | Committed       | Committed Usage    |
+| AWS       | CUR (PurchaseOption)        | Spot Instances     | Dynamic         | Spot               |
+| AWS       | CUR (PurchaseOption)        | Dedicated Hosts    | Standard        | (depends on usage) |
+| Microsoft | Cost Details (PricingModel) | OnDemand           | Standard        |                    |
+| Microsoft | Cost Details (PricingModel) | SavingsPlan        | Committed       | Committed Spend    |
+| Microsoft | Cost Details (PricingModel) | Reservation        | Committed       | Committed Usage    |
+| Microsoft | Cost Details (PricingModel) | Spot               | Reservation     |                    |
```
Comment (flanakin) [inline supporting_content/columns/pricingcategory.md:26]:
```suggestion
| Microsoft | Cost Details (PricingModel) | Spot               | Dynamic         |                    |
```

File: supporting_content/columns/pricingcategory.md [hunk 2]
```diff
@@ -14,68 +14,69 @@ Current column mappings found in available data sets:
...
-- AWS: https://docs.aws.amazon.com/cur/latest/userguide/product-columns.html
-- GCP
-- Microsoft: https://learn.microsoft.com/azure/cost-management-billing/automate/understand-usage-details-fields
+* AWS: https://docs.aws.amazon.com/cur/latest/userguide/product-columns.html
+* GCP
+* Microsoft: https://learn.microsoft.com/azure/cost-management-billing/automate/understand-usage-details-fields
 
 ## Discussion Topics
 
-- Goal of the column is to enable practitioners to identify charges that have reduced prices vs. not.
```
Comment (flanakin) [inline supporting_content/columns/pricingcategory.md:36]: No changes to this section. I just standardized to use `*` instead of `-` for bullets.

**Review by SonalGarg-SG — APPROVED** [review]

**Review by marc-perreaut — APPROVED** [review]
All good!

**Review by udam-f2 — APPROVED** [review]

**Review by mike-finopsorg — APPROVED** [review]

**Comment by jpradocueva** [comment]
Approved by the group on April 18

**Pull request merged**